library-scripts/shellcheck-centos7.sh

```diff
@@ -0,0 +1,3 @@
+
+
```

Comment:
just a note here that the container's user does not have sudo command available to them yet, so it is not as easy as it looks (just running the below command, actually fails atm)